app/services/create_rsf.rb

```diff
@@ -0,0 +1,19 @@
+class CreateRsf
+  attr_reader :payload, :register_name
+
+  def initialize(payload, register_name)
+    @payload = payload
+    @register_name = register_name
+  end
+
+  def call
```

Comment:
is `call` a Ruby convention or does it have a special meaning e.g. I have seen it used by `lambda`?

Reply:
This service object is only responsible for one thing so `call` seemed relevant.